app/services/code_climate/api/object.rb

```diff
@@ -0,0 +1,13 @@
+module CodeClimate
+  module Api
+    class Object
```

Comment:
I would try to find another name for this class, Ruby already has a class called Object, seems odd for me

Reply:
Yes, I didn't know what to use. `CodeClimate::Api::Resource` could be better?

Reply:
Changed it to Resource